src/mpyl/project.py

```diff
@@ -269,6 +271,13 @@ def from_config(values: dict):
         props = values.get('properties')
         kubernetes = values.get('kubernetes')
         traefik = values.get('traefik')
+        pr_number = os.getenv('CHANGE_ID')
+
+        if traefik and pr_number:
+            for hosts in traefik['hosts']:
+                if 'pr' in hosts['host']:
+                    hosts['host']['pr'] = re.sub('{PR-NUMBER}', pr_number, hosts['host']['pr'])
+
```

Comment:
We should also replace the env vars which contain {namespace} in Properties.from_config(props) with `namespace`. See e.g. https://github.com/Vandebron/Vandebron/blob/6a0a8750830417f9ba02a177716ff98342bf3057/energy-trading/trades/retrieve-market-results-job/deployment/project.yml for a project yaml with template {namespace} urls